util/batch/batch.js

```diff
@@ -216,7 +216,8 @@ steal('can/util/can.js', function (can) {
 			// Don't send events if initalizing.
 			if (!item._init) {
 				event = typeof event === 'string' ? {
-					type: event
+					type: event,
+					batchNum: can.batch.batchNum
```

Comment:
@justinbmeyer Am I right to assume that this is a reasonable default? It seemed correct based on how I understand `can.batch.batchNum` to work. I wondered if it should be added to the other `event` objects created [here](https://github.com/bitovi/canjs/blob/master/util/jquery/jquery.js#L16-L20) and [here](https://github.com/bitovi/canjs/blob/master/event/event.js#L241-L245) also.